app/validators/empty_profile_field_names_validator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class EmptyProfileFieldNamesValidator < ActiveModel::Validator
+  def validate(account)
+    return if account.fields.empty?
+
+    account.errors.add(:fields, 'Names of profile fields cannot be empty') if fields_with_values_missing_names?(account)
```

Comment:
We most likely want a translatable string here, since it is user-facing!

Reply:
Thanks for reviewing! I have updated the validator to use a translatable message and added that to the `en` translations YML file, but I'm not clear on how that translation gets into all the other locales. Is there a process for that I'm just not finding in the dev setup docs?

Reply:
No, what you did is enough! It'll get picked up by the translation tool.